docs/application/dotnet/getting-started/sec-privileges.md

```diff
@@ -0,0 +1,191 @@
+| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used, the IMEI, and the statuses of calls. |
+| `http://tizen.org/privilege/telephony.admin` | platform |  | The application can manage telephony settings, such as those for incoming and outgoing calls, forwarding and holding calls, networks, and SIM cards. |
+| `http://tizen.org/privilege/tethering.admin` | platform |  | The application can enable and disable tethering services. This may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/use_ir` | public |  | The application can use the infrared transmitter. |
+| `http://tizen.org/privilege/voicecontrol.manager` | platform |  | The application can record user's voice and recognise it so that voice commands can be used to control this app. It can also give responses to user's commands. |
+| `http://tizen.org/privilege/volume.set` | public |  | The application can adjust the volume for different features, such as notification alerts, ringtones, and media. |
+| `http://tizen.org/privilege/vpnservice` | public |  | The application can manage the VPN (virtual private network) and change its settings. |
+| `http://tizen.org/privilege/web-history.admin` | platform | User history | The application can manage user's Internet history. |
+| `http://tizen.org/privilege/widget.viewer` | public |  | The application can show widgets, and information from their associated applications, on the home screen. |
+| `http://tizen.org/privilege/wifidirect` | public |  | The application can enable and disable Wi-Fi Direct, manage Wi-Fi Direct connections, and change Wi-Fi Direct settings. |
+| `http://tizen.org/privilege/window.priority.set` | public |  | The application can appear on top of other windows and screens, including the lock screen, according to the order of priority of the windows. This may prevent user from interacting with other applications or screens until the window for this application is closed. |
+| `http://tizen.org/privilege/zigbee` | public |  | The application can connect a ZigBee coordinator to end devices and control connected end devices. |
+| `http://tizen.org/privilege/zigbee.admin` | platform |  | The application can control a connected ZigBee coordinator, e.g. turning it on or off. |
+
+<a name="nonAPI"></a>
+## Non-API Bound Privileges
+
+
+Tizen application privileges are loosely bound to APIs, so most of the
+privileges can be identified by the APIs that the application calls.
+However, there are some privileges that are not coupled with the Tizen
+APIs. To allow easy identification, those privileges are mapped to
+corresponding system resources - same as other privileges.
```

Comment:
corresponding system resources that are similar as other privileges.

Reply:
done